docker-compose.yml

```diff
@@ -0,0 +1,82 @@
+version: '3.4'
+
+configs:
+  prometheus_config:
+    file: ./TodoApi/prometheus.yml
+
+services:
+  todoapi:
+    image: ${DOCKER_REGISTRY-}todoapi
+    build:
+      context: .
+      dockerfile: TodoApi/Dockerfile
+    environment:
+      - ASPNETCORE_ENVIRONMENT=Development
+      - ASPNETCORE_URLS=https://+:443;http://+:80
+      - OTEL_EXPORTER_OTLP_ENDPOINT=http://jaeger:4317
+    ports:
+      - "5000:80"
+      - "5001:443"
+    volumes:
+      - ${APPDATA}/Microsoft/UserSecrets:/root/.microsoft/usersecrets:ro
+      - ${APPDATA}/ASP.NET/Https:/root/.aspnet/https:ro
+      - type: bind
+        source: ./TodoApi/Todos.db
+        target: /app/Todos.db
+      - type: bind
+        source: ./TodoApi/Todos.db-shm
+        target: /app/Todos.db-shm
+      - type: bind
+        source: ./TodoApi/Todos.db-wal
+        target: /app/Todos.db-wal
```

Comment:
This seems strange, mapping the sqlite db into the container. Maybe we should move this into a folder and map the entire folder instead.

Reply:
Yep, I found it strange too. I was trying to avoid moving it to a folder. I will do it.

Reply:
Done. I've changed the database to a folder called `.db` and mapped it in the docker-compose file.